bodhi/server/models.py

```diff
@@ -2430,8 +2432,6 @@ def edit(cls, request, data):
 
         # And, updates with new or removed builds always get their karma reset.
         # https://github.com/fedora-infra/bodhi/issues/511
```

Comment:
This comment is not relevant here any more, we should probably move it with `data['karma_critipath'] =0` line.

Reply:
@karmadolkar Just this left ^^. We should either remove that comment or move it with the code.